### DISCUSSION TITLE
The domain registered in the system as school issued email address isn't actually issued by our school

### DISCUSSION THREAD

**Discussion by hexaorzo** [community/community#63085] [GitHub Education]
### Hi everyone,

I'm unable to renew my Student developer pack using picture of my ID card.
![image](https://github.com/community/community/assets/67504410/d5c51a4d-9383-48d9-a672-81df50731ec0)

It shows that khabri.tech (screenshot the website is attached below) is a domain for academic email addresses for our school Little Flower School (https://www.littleflowerschoolgkp.com) which is not correct as our school does not provides any academic email address to its students.

![image](https://github.com/community/community/assets/67504410/66ff15a7-42ae-405f-9e66-87de13c80643)

![image](https://github.com/community/community/assets/67504410/43eb5ae0-abac-41fa-a7f6-caa66b4d89e9)

**Comment by javiermm8** [comment]
I've got the same problem. And my school don't provide emails for the students.

**Comment by humairshoukat** [comment]
No issue submit your Student ID Card with clear expiration date mentioned on it.

**Reply by hexaorzo** [reply]
I have tried that but no luck

**Reply by javiermm8** [reply]
Try taking  the foto from your phone, not uploading it from your gallery. It works for me.

**Reply by hexaorzo** [reply]
I've tried that too. My problem is apparently totally different from the problem you had faced.

**Comment by ghost** [comment]
I've got the same problem. GitHub is not even replying to email regarding this.

**Reply by javiermm8** [reply]
Try contact a GitHub Campus Expert and ask him about this issue. ¯\_(ツ)_/¯
